doc/conf.py

```diff
@@ -434,6 +434,18 @@ def _str_examples(self):
 }
 html_show_sourcelink = False
 
+
+def get_version_match(semver):
+    """Evaluate the version match for the multi-documentation."""
+    if semver.endswith("dev0"):
+        return "dev"
+    major, minor, _ = semver.split(".")
+    return ".".join([major, minor])
+
+
+cname = os.getenv("DOCUMENTATION_CNAME", "<DEFAULT_CNAME>")
```

Comment:
We usually place this variable close to the "project variables", at the top of the `conf.py` file.